core/services/keeper/upkeep_executer.go

```diff
@@ -309,3 +294,31 @@ func (ex *UpkeepExecuter) turnBlockHashBinary(registry Registry, head *evmtypes.
 	binaryString := fmt.Sprintf("%b", hashAtHeight.Big())
 	return binaryString, nil
 }
+
+func buildJobSpec(
+	jb job.Job,
+	upkeep UpkeepRegistration,
+	ormConfig RegistryGasChecker,
+	exConfig RegistryGasChecker,
+	gasPrice *big.Int,
+	gasTipCap *big.Int,
+	gasFeeCap *big.Int,
+	chainID string,
+) map[string]interface{} {
+	return map[string]interface{}{
+		"jobSpec": map[string]interface{}{
+			"jobID":                 jb.ID,
+			"fromAddress":           upkeep.Registry.FromAddress.String(),
+			"contractAddress":       upkeep.Registry.ContractAddress.String(),
+			"upkeepID":              upkeep.UpkeepID.String(),
+			"prettyID":              upkeep.PrettyID(),
+			"performUpkeepGasLimit": upkeep.ExecuteGas + ormConfig.KeeperRegistryPerformGasOverhead(),
+			"checkUpkeepGasLimit": exConfig.KeeperRegistryCheckGasOverhead() + uint64(upkeep.Registry.CheckGas) +
+				exConfig.KeeperRegistryPerformGasOverhead() + upkeep.ExecuteGas,
```

Comment:
This is an issue in previous code too, but we should use the same `KeeperRegistryPerformGasOverhead` in both places, not sure why we have this unnecessary duplication

Reply:
It is confusing for sure. What would be the scope to fix the problem?

Reply:
I think we should remove it from one of these configs, can have that as a separate ticket

Reply:
Could you create a ticket for cleaning this up?